llvm/include/llvm/Target/TargetLoweringObjectFile.h

```diff
@@ -305,6 +305,10 @@ class LLVM_ABI TargetLoweringObjectFile : public MCObjectFileInfo {
     return nullptr;
   }
 
+  virtual bool canEmitConstantPtrAuthAsIRelative(const Constant *CPA) const {
```

Comment:
I've kept this as `Constant` instead of `ConstantPtrAuth` to avoid having additional includes in this file.